doc_src/en/HowTo_UseTM.xml

```diff
@@ -74,21 +74,30 @@
 		linkend="project.folder.source" endterm="project.folder.source.title"/>
 		folder.</para>
 		
-		<para>After reloading the old project, all its translation memory will
-		be applied to the new source files.</para>
-
 		<note>
 		  <para>That’s the easiest way to work on the new version of an already
 		  translated document.</para>
 		</note>
+
+		<para>After reloading the old project, its translation memory will
+		be applied to the new source files. Identical segments will be
+		automatically translated. Similar segments will be provided with
+		matches from the project translated memory as you enter them.</para>
+
 	  </listitem>
 	  
 	  <listitem>
-		<para>Put the reference memories that you took from the old project and
-		put them in the <link linkend="project.folder.tm"
-		endterm="project.folder.tm.title"/> folder of the new project. Depending
-		on how reliable a given memory is, you might prefer to put it in one of the
-		following subfolders:</para>
+		<para>Alternatively, put the reference memories from the old project in
+		the <link linkend="project.folder.tm" endterm="project.folder.tm.title"/>
+		folder of the new one.</para>
+
+		<note>
+		  <para>That’s the preferred method if you want to start your
+		  translation from scratch.</para>
+		</note>
+
+		<para>Depending on how reliable the memory is, you can prefer to put it
+		in either of the following subfolder:</para>
```

Comment:
Last look:

Change "in either of the following subfolder:" to "in one of the following subfolders:"